README.md

```diff
@@ -1,230 +1,41 @@
-
-echo 'export LD_LIBRARY_PATH=$PDC_DIR/lib:$LD_LIBRARY_PATH' >> $WORK_SPACE/pdc_env.sh
-echo 'export PATH=$PDC_DIR/include:$PDC_DIR/lib:$PATH' >> $WORK_SPACE/pdc_env.sh
-```
-
-## About Spack
-
-One can also install `PDC` with [`Spack`](https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/pdc/package.py), with which the dependencies of `PDC` can be easily managed and installed.
-
-```bash
-git clone -c feature.manyFiles=true https://github.com/spack/spack.git
-cd spack/bin
-./spack install pdc
-```
-
-## Running PDC
-
-Essentially, PDC is a typical client-server application.
-To run `PDC`, one needs to start the server processes first, and then the clients can be started to issue RPC requests handled by the `Mercury` RPC framework. 
-
-We provide [`mpi_test.sh` utility](https://github.com/hpc-io/pdc/blob/develop/examples/mpi_test.sh) for running MPI tests. For example, on a regular Linux machine, you may run the following:
-
-```bash
-export JOB_RUNNER=mpiexec
-cd $PDC_DIR/bin
-./mpi_test.sh ./pdc_init $JOB_RUNNER 2 4
-```
-
-This is test will start 2 processes for PDC servers. The client program ./pdc_init will start 4 processes. Similarly, one can run any of the client examples in `ctest`.
-
-Depending on the specific HPC environment where you run `PDC` , the value of `$JOB_RUNNER` variable can be changed to `srun` (for NERSC), `aprun` (for Theta), or `jsrun` for `Summit`, accordingly.
-
-These source code will provide some knowledge of how to use PDC. For more reference, one may check the documentation folder in this repository.
-
-# PDC on Cori
-
-If you are running `PDC` on Cori supercomputer, here are some tips you would need to follow:
-
-* On Cori, it is recommended to use `cc` as the default compiler when compiling PDC and its dependencies. 
-* When preparing compilation for `PDC` using `CMake`, it is suggested to append console argument `-DMPI_RUN_CMD=srun` so that `ctest` can be executed on Cori.
-* Sometimes, it might be helpful to unload `darshan` module before the installation. 
-
-* For opening an interactive job session on Cori, it is recommended to add `--gres=craynetwork:2` option to the `salloc` command:
-    ```bash
-    salloc -C haswell -N 4 -t 01:00:00 -q interactive --gres=craynetwork:2
-    ```
-* To launch the PDC server and the client, add `--gres=craynetwork:1` before the executables, for example:
-
-  * Run 4 server processes, each on one node in background:
-    ```bash
-    srun -N 4 -n  4 -c 2 --mem=25600 --cpu_bind=cores --gres=craynetwork:1 --overlap ./bin/pdc_server.exe &
-    ```
-
-  * Run 64 client processes that concurrently create 1000 objects in total:
-    ```bash
-    srun -N 4 -n 64 -c 2 --mem=25600 --cpu_bind=cores --gres=craynetwork:1 --overlap ./bin/create_obj_scale -r 1000
-    ```
-
-
-
+You can access the full documentation of PDC installation instructions and examples at [pdc.readthedocs.io](https://pdc.readthedocs.io/en/code-docs)
```

Comment:
Suggested text:
"Full documentation of PDC with installation instructions, code examples for using PDC API, and research publications are available at [pdc.readthedocs.io](https://pdc.readthedocs.io/en/code-docs).

Reply:
Is this going to be in the /code-docs folder or the main?

Reply:
ops, main, let me fix that